src/Forms/TrackProgressForm.cs

```diff
@@ -199,6 +199,10 @@ public void AppendTextBox(string value)
                     ProcessLogsRichTextBox.SelectionColor = Color.White;
 
                 ProcessLogsRichTextBox.AppendText(Environment.NewLine + $"--> {value}");
+
+                // Scroll to the end
+                ProcessLogsRichTextBox.SelectionStart = ProcessLogsRichTextBox.TextLength;
+                ProcessLogsRichTextBox.ScrollToCaret();

```

Comment:
Was this auto-scroll request made by Ankit?

Reply:
It was made during the self-host of AME-Copilot by Andy so that the user doesn't have to scroll the ricktextbox to view the new updates. The user would always see from the end of the box, as the updates keep getting appended.